src/web3mail/sendEmail.ts

```diff
@@ -141,11 +141,31 @@ export const sendEmail = async ({
     const emailSubjectId = generateSecureUniqueId(16);
     const emailContentId = generateSecureUniqueId(16);
     const optionsId = generateSecureUniqueId(16);
+
     await iexec.secrets.pushRequesterSecret(emailSubjectId, vEmailSubject);
-    await iexec.secrets.pushRequesterSecret(emailContentId, vEmailContent);
+    // create AES encryptionKey & encrypt content
+    const emailContentEncryptionKey = iexec.dataset.generateEncryptionKey();
+    const encryptedFile = await iexec.dataset
+      .encrypt(Buffer.from(vEmailContent, 'utf8'), emailContentEncryptionKey)
+      .catch((e) => {
+        throw new WorkflowError('Failed to encrypt e-mail content', e);
+      });
+    // upload encrypted content on IPFS
+    const cid = await ipfs.add(encryptedFile).catch((e) => {
+      throw new WorkflowError('Failed to upload encrypted e-mail content', e);
+    });
```

Comment:
while `e-mail` and `email` are both correct orthographs, `e-mail` seems to be dated, and most modern apps use `email` instead.

```suggestion
    const encryptedFile = await iexec.dataset
      .encrypt(Buffer.from(vEmailContent, 'utf8'), emailContentEncryptionKey)
      .catch((e) => {
        throw new WorkflowError('Failed to encrypt email content', e);
      });
    // upload encrypted content on IPFS
    const cid = await ipfs.add(encryptedFile).catch((e) => {
      throw new WorkflowError('Failed to upload encrypted email content', e);
    });
```